src/com/nutrons/nu17/subsystems/Drivetrain.java

```diff
@@ -1,104 +1,115 @@
 package com.nutrons.nu17.subsystems;
 
+import com.nutrons.nu17.OI;
 import com.nutrons.nu17.RobotMap;
+
+import edu.wpi.first.wpilibj.AnalogGyro;
 import edu.wpi.first.wpilibj.Encoder;
 import edu.wpi.first.wpilibj.PIDController;
-import edu.wpi.first.wpilibj.PIDOutput;
-import edu.wpi.first.wpilibj.PIDSource;
 import edu.wpi.first.wpilibj.PIDSourceType;
+import edu.wpi.first.wpilibj.RobotDrive;
 import edu.wpi.first.wpilibj.Talon;
 import edu.wpi.first.wpilibj.command.Subsystem;
+import lib.EncoderWrapper;
+import lib.GyroWrapper;
+import lib.HoldPID;
 
+/**
+ * Drivetrain code four a 4 wheel drive
+ */
 public class Drivetrain extends Subsystem {
 	
-	private final Talon FRONT_LEFT_WHEEL = new Talon(RobotMap.FRONT_LEFT);
-	private final Talon BACK_LEFT_WHEEL = new Talon(RobotMap.FRONT_RIGHT);
-	private final Talon FRONT_RIGHT_WHEEL = new Talon(RobotMap.BACK_LEFT);
-	private final Talon BACK_RIGHT_WHEEL = new Talon(RobotMap.BACK_RIGHT);
-	
-	private final Encoder ENCODER_1 = new Encoder(
-			RobotMap.DT_ENCODER_1, 
-			RobotMap.DT_ENCODER_2);
-	private final Encoder ENCODER_2 = new Encoder(
-			RobotMap.DT_ENCODER_3, 
-			RobotMap.DT_ENCODER_4);
-
-	// TODO: tune these constants
-	private static final double P_DISTANCE = 0.025;
-	private static final double I_DISTANCE = 0.0;
-	private static final double D_DISTANCE = 0.01;
-
-	private final PIDController DISTANCE_PID = new PIDController(
-			P_DISTANCE, 
-			I_DISTANCE, 
-			D_DISTANCE,
-			new DriveSourcePID(), 
-			new DriveOutputPID());
-
 	public Drivetrain() {
-		ENCODER_1.setDistancePerPulse(1);
-		ENCODER_2.setDistancePerPulse(1);
-	}
-
-	public void initDefaultCommand() {
 		//empty
 	}
+	
+	// Motors
+		public Talon leftDriveA = new Talon(RobotMap.LEFT_DRIVE_MOTOR_A);
+		public Talon leftDriveB = new Talon(RobotMap.LEFT_DRIVE_MOTOR_B);
+		public Talon rightDriveA = new Talon(RobotMap.RIGHT_DRIVE_MOTOR_A);
+		public Talon rightDriveB = new Talon(RobotMap.RIGHT_DRIVE_MOTOR_B);
+    
+	// Sensors
```

Comment:
Fix the indentation here.  The declarations should be indented to the same level as the comment.